sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobUrlParts.java

```diff
@@ -166,24 +167,24 @@ public BlobUrlParts setSnapshot(String snapshot) {
     }
 
     /**
-     * Gets the {@link BlobServiceSasQueryParameters} representing the SAS query parameters that will be used to
+     * Gets the {@link CommonSasQueryParameters} representing the SAS query parameters that will be used to
      * generate the SAS token for this URL.
      *
-     * @return the {@link BlobServiceSasQueryParameters} of the URL
+     * @return the {@link CommonSasQueryParameters} of the URL
      */
-    public BlobServiceSasQueryParameters getSasQueryParameters() {
```

Comment:
Was this API in existence before 12.0.0 or 12.0.1? If so we will want to mark this one as deprecated and add the `CommonSasQueryParameters` variants as new APIs.

Reply:
If this is needed to be done creating an internal method to convert from `BlobServiceSasQueryParameters` to `CommonSasQueryParameters` and vice-versa will make the internal implementation simpler.

Reply:
I dont think it would be possible to convert between the two since CommonSASQueryParameters is a super set of the BlobServiceSasQueryParameters (if its an account sas being used the blob service sas would be invalid) maybe we can discuss the right way to deal with this.

Reply:
I guess at the very least we can pretend like its a BlobServiceSasQueryParameter that way we can mark it as deprecated and remove it in the future

Reply:
@alzimmermsft Can you check to see if this is better? I brought back the old methods and deprecated them.